src/main/java/redis/clients/jedis/MultiNodePipelineBase.java

```diff
@@ -47,6 +53,9 @@ public abstract class MultiNodePipelineBase implements PipelineCommands, Pipelin
   private final CommandObjects commandObjects;
   private GraphCommandObjects graphCommandObjects;
 
+  private final ExecutorService executorService = new ThreadPoolExecutor(3, Integer.MAX_VALUE, 60L, TimeUnit.SECONDS,
+          new ArrayBlockingQueue<>(3));
```

Comment:
- `Integer.MAX_VALUE` for maximumPoolSize seems risky.
- `60L, TimeUnit.SECONDS` - I understand it's a lot for most applications but still small enough. We shouldn't restrict it in smaller time as we are not allowing users to modify yet.

Reply:
How about simply [Executors#newFixedThreadPool](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Executors.html#newFixedThreadPool-int-)

Reply:
Theoretically, the maximum number of nodes is 16384.redis. The author recommends that it be within 1000. We set it to 1024. It is more suitable. What do you think?

Reply:
> * `Integer.MAX_VALUE` for maximumPoolSize seems risky.
> * `60L, TimeUnit.SECONDS` - I understand it's a lot for most applications but still small enough. We shouldn't restrict it in smaller time as we are not allowing users to modify yet.

provide default values that can be customized by users， what do you think?
![image](https://user-images.githubusercontent.com/10794822/227928167-84f24c73-ab54-4968-8f36-18fbb2585b91.png)

Reply:
Our (alibaba) programming specification does not recommend using `Executors.*` to directly create a thread pool. The correct way is to use `ThreadPoolExecutor`, see [here](https://github.com/alibaba/Alibaba-Java-Coding-Guidelines#concurrency) for details.

Again, I think we have to provide `JedisThreadFactory` to name the threads (Jedis may introduce other ThreadPools in the future, we can reuse JedisThreadFactory).

Also, I think the executors in `MultiNodePipelineBase` need to have modification interfaces (we don't have to create them when users want to pass their own thread pool in).